src/features/pages/home/live-pricing/table-component/live-market-table.tsx

```diff
@@ -0,0 +1,158 @@
+                    const selected_market_data = res[item]
+                    const result = Object.values(selected_market_data)
+                    setMarketsData(result)
+                }
+            })
+        }
+    }, [rawMarketsData, selected_market])
+
+    useEffect(() => {
+        updateData()
+    }, [updateData])
+
+    const columns = useLiveColumns()
+
+    const table = useReactTable({
+        data: markets_data,
+        columns,
+        state: {
+            sorting,
+        },
+        getCoreRowModel: getCoreRowModel(),
+        getSortedRowModel: getSortedRowModel(),
+        onSortingChange: setSorting,
+    })
+
+    const rows = table.getRowModel().rows.slice(0, TABLE_VISIBLE_ROWS)
+
+    return (
+        <>
+            <Flex.Box justify="center" mt="16x">
+                <table>
+                    <thead>
+                        {table.getHeaderGroups().map((headerGroup) => (
+                            <Flex.Box key={headerGroup.id} justify="center">
+                                <tr className={table_row_header}>
+                                    {headerGroup.headers.map((header) => (
+                                        <th key={header.id}>
+                                            {header.isPlaceholder
+                                                ? null
+                                                : flexRender(
+                                                      header.column.columnDef.header,
+                                                      header.getContext(),
+                                                  )}
+                                        </th>
+                                    ))}
+                                </tr>
+                            </Flex.Box>
+                        ))}
+                    </thead>
+                    <tbody>
+                        {rows.map((row) => (
+                            <Flex.Box key={row.id} justify="center">
+                                <tr className={table_row_data}>
+                                    {row.getVisibleCells().map((cell) => (
+                                        <td key={cell.id}>
+                                            {flexRender(
+                                                cell.column.columnDef.cell,
+                                                cell.getContext(),
+                                            )}
+                                        </td>
+                                    ))}
+                                </tr>
+                            </Flex.Box>
+                        ))}
+                    </tbody>
+                </table>
+            </Flex.Box>
+
+            <Flex.Box justify="center" align="center" mt="18x" gap="10x">
+                <Typography.Paragraph size="medium">
+                    <Localize
+                        translate_text="_t_<0>See all {{display_name}} market ></0>_t_"
```

Comment:
Check your crowdin/messages.json, it's never translate
We have to pass all text instead